test/util.ts

```diff
@@ -1,5 +1,5 @@
 import type { Token } from '../src/core/types'
-import { tokens } from '../dex-config.example.json'
+import { tokens } from '../dex-config.json'
```

Comment:
Tests should rely on a fixed data, because snapshots might include these tokens data. `dex-config.example.json` is fixed and immutable, `dex-config.json` - absolutely not.

Reply:
I didn't intend to make this change, it happened by accident. Reverted the change.